Integrated-Windows-Auth-Flow/pom.xml

```diff
@@ -11,16 +11,24 @@
         <dependency>
             <groupId>com.microsoft.azure</groupId>
             <artifactId>msal4j</artifactId>
-            <version>1.2.0</version>
+            <version>1.4.0</version>
         </dependency>
         <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-simple</artifactId>
             <version>1.7.29</version>
-
         </dependency>
     </dependencies>
+
     <build>
+        <resources>
```

Comment:
It would be better to put .json files to "src/main/resources" folder , according to standard directory layout for maven project - https://maven.apache.org/guides/introduction/introduction-to-the-standard-directory-layout.html

Reply:
Made the change, the .json is now pulled in automatically without being hardcoded in the .pom